estate/views/estate_property_offer.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="estate_property_offer_action" model="ir.actions.act_window">
+        <field name="name">Property offer</field>
+        <field name="res_model">estate.property.offer</field>
+        <field name="view_mode">tree,form</field>
+
+    </record>
+
+    <record id="estate_property_offer_tree" model="ir.ui.view">
+        <field name="name">estate.property.offer.tree</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <tree editable="bottom" decoration-danger="status == 'refused'"
+                decoration-success="status == 'accepted'">
+                <field name="price" />
+                <field name="status" />
+                <field name="partner_id" />
+                <field name="property_id" />
+                <field name="date_deadline" />
+                <field name="validity" />
+                <button name="action_accept" type="object" string="Accept" icon="fa-check"
+                    invisible="status in ['accepted','refused']" />
+                <button name="action_refuse" type="object" string="Refuse" icon="fa-times"
+                    invisible="status in ['accepted','refused']" />
```

Comment:
When we have many attributes it's better to make a multiple line with them, leave like this for now,